_includes/package_readme.html

```diff
@@ -0,0 +1,12 @@
+{% capture readme %}
+    {% include babel/packages/{{ page.package }}/README.md %}
+{% endcapture %}
+
+{{ readme
+    | newline_to_br
+    | split: "<br />"
+    | slice: 4, 1000
```

Comment:
I also can use `shift` there.

Like this : 
```
    | split: "<br />"
    | shift
    | shift
    | shift
    | shift
    | join: "<br />"
```

This way I don't need to limit the string to 1000 but I don't like it.
Do you mind @thejameskyle ?